my-youtube/src/app/search/search-item/search-item.model.ts

```diff
@@ -0,0 +1,56 @@
+export interface IItem {
```

Comment:
naming interfaces starting with I is not recommended, but you can do it if you really want to